readme.md

```diff
@@ -1,2 +1,5 @@
 # JP App
 Learn Japanese with the help of AI!
+---
```

Comment:
Please leave a new line before '---' as the line before this is set to a header.